### DISCUSSION TITLE
[RELEASE] One Room

### DISCUSSION THREAD

**Discussion by TaborScott** [nzp-team/nzportable#844] [Map Releases]
A rushed, terrible version of the One Room Challenge.
Any bugs/glitches found report on the discussion... :)
If there's any problems with setup let me know!

![OneRoom](https://github.com/user-attachments/assets/7cda7672-8ff7-460d-adba-37e1b1443337)

[Download](https://archive.org/download/one-room/OneRoom.zip)

FUTURE PLANS:
CUSTOM SKINS
END GAME BUTTON
MUSIC
CUSTOM GUN SOUNDS
REVAMP OF START ROOM

**Comment by TaborScott** [comment]
UPDATE: 
ZOMBIE HUDDLING FIXED
MAP SMALLER

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the archive is not equivalent to the gamedir root: https://github.com/nzp-team/nzportable/discussions/159
